pyproject.toml

```diff
@@ -86,6 +86,7 @@ tests = [
     "pytest-cov>=2.6.1",
     "coverage>=5.1",
     "pytest-benchmark",
+    "pytest-mock",
```

Comment:
Is unittest mocking stuff not good enough?

Reply:
not good enough

Reply:
Can we test some other way then?